package.json

```diff
@@ -10,6 +10,8 @@
     "license": "Apache-2.0",
     "dependencies": {
         "@craco/craco": "^6.4.5",
+        "@ethersproject/transactions": "^5.7.0",
```

Comment:
Why do we need this dependency ? Is it to "force" a specific version ?

Reply:
No, I forgot to remove it after some tests. Thanks for spotting it!